code/modules/mob/living/carbon/human/human_update_icons.dm

```diff
@@ -584,7 +591,8 @@ generate/load female uniform sprites matching all previously decided variables
 	else
 		. += "-not_coloured"
 
-	. += "-[gender]"

```

Comment:
is this a fix? gender should be "male" or "female" or "other", which would be different than f/m

Reply:
oop